.travis.yml

```diff
@@ -1,18 +1,16 @@
 language: ruby
 rvm:
   - jruby
-  - 2.0.0
-  - 2.1
-  - 2.2
-  - 2.3.0
+  - 2.1.10
+  - 2.2.6
+  - 2.3.3
```

Comment:
I think we should use the latest patch version possible. I don't see any point in testing on the older versions. Also, Ruby 2.0.0 is not supported anymore, so it's a good idea to specify the latest patch version. This PR should be merged, Ruby versions for `.travis.yml` can be discussed separately.